sql/src/main/java/io/crate/operation/reference/sys/node/NodeOsInfoExpression.java

```diff
@@ -43,6 +48,25 @@ public Integer value() {
                 return info.availableProcessors();
             }
         });
+        childImplementations.put(OS_NAME, new OsInfoExpression() {
+            @Override
+            public String value() {
+                return Constants.OS_NAME;
+            }
+        });
+        childImplementations.put(OS_ARCH, new OsInfoExpression() {
+            @Override
+            public String value() {
+                return Constants.OS_ARCH;
+            }
+        });
+        childImplementations.put(OS_VERSION, new OsInfoExpression() {
+            @Override
+            public String value() {
```

Comment:
@mfussenegger in es 2.0 already in but with String data type https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/monitor/os/OsProbe.java#L130